build.sbt

```diff
@@ -62,17 +60,17 @@ val `faithful-cats` =
     .settings(
       crossScalaVersions := crossScalaV,
       libraryDependencies ++= Seq(
-        "org.typelevel" %%% "cats-core" % "2.0.0",
-        "org.typelevel" %%% "cats-laws" % "2.0.0" % Test,
-        "org.typelevel" %%% "discipline-scalatest" % "1.0.0-M1" % Test,
-        "org.scalatest" %%% "scalatest" % "3.1.0-M2" % Test
+        "org.typelevel" %%% "cats-core" % "2.1.1",
+        "org.typelevel" %%% "cats-laws" % "2.1.1" % Test,
+        "org.typelevel" %%% "discipline-scalatest" % "1.0.1" % Test,
+        "org.scalatest" %%% "scalatest" % "3.2.0-M4" % Test
```

Comment:
Same here: what do you think about using 3.1.x?

Reply:
Yes, down to `3.1.2` now